src/overlays/actors/ovl_En_Go/z_en_go.c

```diff
@@ -1967,32 +2551,35 @@ void func_80A15B80(EnGo* this, PlayState* play) {
     Matrix_RotateXS(this->actor.shape.rot.x, MTXMODE_APPLY);
 
     gSPMatrix(POLY_OPA_DISP++, Matrix_NewMtx(play->state.gfxCtx), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
-    gSPDisplayList(POLY_OPA_DISP++, (this->unk_390 & 0x100) ? gGoronSnowballDL : gGoronRolledUpDL);
+    gSPDisplayList(POLY_OPA_DISP++, (this->actionFlags & ENGO_FLAG_SNOWBALLED) ? gGoronSnowballDL : gGoronRolledUpDL);
 
     CLOSE_DISPS(play->state.gfxCtx);
 }
 
+/**
+ * OverrideLimbDrawOpa function for the Gorons
+ */
 s32 EnGo_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx) {
     EnGo* this = THIS;
-    Vec3f sp30;
+    Vec3f worldPos;
     s32 idx;
```

Comment:
```suggestion
    s32 rotTableIndex;
```
There are a couple indicies here, so I would be a little more specific.